src/musicPlayer.java

```diff
@@ -0,0 +1,37 @@
+import components.queue.Queue;
+import components.queue.Queue1L;
+
+/** */
+public class musicPlayer {
```

Comment:
VSCode should flag this, but class names have a slightly different naming convention. Specifically, I would name this `MusicPlayer`.